react/week4/user_search/src/Login.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useContext } from "react";
+import { UserContext } from "./UserContext";
+
+export default function Login({ setUserId }) {
+	const { userId } = useContext(UserContext);
+	const [user, setUser] = useState(userId || "");
+
+	return (
+		<form onSubmit={setUserId(user)}>
```

Comment:
No need for submitting form since the result is related to typing in input and changing the value of it.